ibm/resource_ibm_is_vpc.go

```diff
@@ -350,6 +350,16 @@ func resourceIBMISVPCValidator() *ResourceValidator {
 			MinValueLength:             1,
 			MaxValueLength:             63})
 
+	validateSchema = append(validateSchema,
+		ValidateSchema{
+			Identifier:                 "tag",
+			ValidateFunctionIdentifier: ValidateRegexpLen,
+			Type:                       TypeString,
+			Optional:                   true,
+			Regexp:                     `^[A-Za-z0-9-. _]([A-Za-z0-9-:_ ]*[A-Za-z0-9-_ .])?$`,
```

Comment:
Api Docs has the valid regex for tags.. Please refer this and add regex accordingly
https://cloud.ibm.com/apidocs/tagging#attach-tag